ui/components/DashboardComponent.js

```diff
@@ -1,597 +1,658 @@
+                      .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+                      .join(" ")} adapter on port ${aa.label.split(":")[1]}`
+                }
+              >
+                <Chip
+                  label={aa.label.split(":")[0]}
+                  onClick={self.handleAdapterClick(aa.value)}
+                  icon={logoIcon}
+                  className={classes.chip}
+                  key={`adapters-${ia}`}
+                  variant={isDisabled ? "default" : "outlined"}
+                />
+              </Tooltip>
+            );
+          })}
+        </div>
+      );
+    }
+    let showGrafana;
+    if (grafanaUrl === "") {
+      showGrafana = (
+        <div className={classes.alreadyConfigured}>
+          <Button
+            variant="contained"
+            color="primary"
+            size="large"
+            className={classes.metricsButton}
+            onClick={() => this.handleConfigure("grafana")}
+          >
+            <SettingsIcon className={classes.settingsIcon} />
+            Configure Grafana
+          </Button>
+        </div>
+      );
+    }
+    if (grafana && grafana.grafanaURL && grafana.grafanaURL !== "") {
+      showGrafana = (
+        <Chip
+          label={grafana.grafanaURL}
+          onClick={self.handleGrafanaClick}
+          icon={<img src="/static/img/grafana_icon.svg" className={classes.icon} />}
+          className={classes.chip}
+          key="graf-key"
+          variant="outlined"
+        />
+      );
+    }
+
+    let showPrometheus;
+    if (prometheusUrl === "") {
+      showPrometheus = (
+        <div className={classes.alreadyConfigured}>
+          <Button
+            variant="contained"
+            color="primary"
+            size="large"
+            className={classes.metricsButton}
+            onClick={() => this.handleConfigure("prometheus")}
+          >
+            <SettingsIcon className={classes.settingsIcon} />
+            Configure Prometheus
+          </Button>
+        </div>
+      );
+    }
+    if (prometheus && prometheus.prometheusURL && prometheus.prometheusURL !== "") {
+      showPrometheus = (
+        <Chip
+          label={prometheus.prometheusURL}
+          onClick={self.handlePrometheusClick}
+          icon={<img src="/static/img/prometheus_logo_orange_circle.svg" className={classes.icon} />}
+          className={classes.chip}
+          key="prom-key"
+          variant="outlined"
+        />
+      );
+    }
+
+    const showMetrics = (
+      <div>
+        {showPrometheus}
+        {showGrafana}
+      </div>
+    );
+
+    /**
+     * getMesheryVersionText returs a well formatted version text
+     * @param {string} type type of version could be "latest" or "current"
+     */
+    const getMesheryVersionText = (type) => {
+      const {build, latest, outdated} = this.state.versionDetail
+      if (type === "current") {
+        if (outdated) return `Running Meshery: stable-${build}`

```

Comment:
If the current Meshery version is an edge release then we should display that too, right?

Reply:
@Nikhil-Ladha, good call, yes. @utkarsh-pro, fyi, an Edge release will be in a similar format (e.g. `edge-latest`).

Reply:
Can you please tell me that how an edge release and a stable release is determined? Will it depend on the selected channel?

Reply:
@utkarsh-pro, yes, that's right. Although, currently, the ability to switch between `stable` and `edge` releases is disabled in the UI and must be done separately. We need a doc on this.

Briefly, you can edit your `~/.meshery/meshery.yaml` and on the fourth line update it to `edge-latest`. Then `mesheryctl system restart`.

Reply:
@kumarabd will you speak to how to use environment variables to manipulate stable, edge, version numbers when compiling meshery locally?

Reply:
Should I convert Version struct to this?
```diff
type Version struct {
    Build     string `json:"build,omitempty"`
    Latest   string `json:"latest,omitempty"`
    Outdated  *bool  `json:"outdated,omitempty"`
    CommitSHA string `json:"commitsha,omitempty"`
+   Release string `json:"release,omitempty"
}
```

But I have no idea from where we I can get this `release` info on the server.

Reply:
@kushthedude, can you highlight what changes will need to be done in the build system in order to provide release channel name "stable" or "edge" as an environment variable, so that the `/api/server/version` endpoint can be enhanced to include the channel name?

- Dockerfile: https://github.com/layer5io/meshery/blob/3abaa633b14e331187396f32d36c7d5f1f55b989/Dockerfile#L3-L4
- GitHub action: https://github.com/layer5io/meshery/blob/3abaa633b14e331187396f32d36c7d5f1f55b989/.github/workflows/build-and-release.yml#L63

Reply:
> @kumarabd will you speak to how to use environment variables to manipulate stable, edge, version numbers when compiling meshery locally?

@leecalcote It is controlled by the ld-flags during compile time of the Golang backend server. The flags are available in here - https://github.com/layer5io/meshery/blob/master/Dockerfile#L9. If we have to build multiple images with the different version tags on each build, one way of doing it is to pass docker build argument in the CI itself (Thus not having dependency on the project). This can be done by passing these arguments - https://github.com/layer5io/meshery/blob/master/Dockerfile#L2-L4

Reply:
@utkarsh-pro, yes, adding a new attribute to the Version struct makes sense to me. "Channel" is an ideal name for the attribute.